lib/internal/bootstrap/pre_execution.js

```diff
@@ -245,6 +245,16 @@ function setupFetch() {
         throw new ERR_WEBASSEMBLY_RESPONSE('body has already been used');
       }
 
+      const handler = require('internal/v8').getWasmStreamingHandler();
+      const cached = await handler?.get(response.url);
+      if (cached !== undefined && cache !== null) {
+        if (streamState.setCompiledModuleBytes(cached)) {
+          return;
+        } else {
+          await handler.delete(response.url);
```

Comment:
In this case, V8 has decided not to use the compiled code. That can happen, for example, when running node with certain V8 flags. However, actively deleting something from the cache means that two Node.js processes running with different command-line flags but using the same WebAssembly cache will keep filling and erasing the cache.

Reply:
@tniessen that's a really good point. Do you see a way to isolate this deletion?